alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/fixer/solr/AbstractSolrNodeFixerPlugin.java

```diff
@@ -42,14 +42,14 @@
             NodeIndexHealthReport endpointHealthReport);
 
     protected NodeFixReport trySendSolrCommand(NodeHealthReport unhealthyReport,
-            NodeIndexHealthReport endpointHealthReport, SolrNodeCommand command) {
+            NodeIndexHealthReport endpointHealthReport, SolrNodeCommand command, boolean targetTransaction) {
```

Comment:
Instead of adding an additional parameter, can you put it in the `SolrNodeCommand` enum?

You can add an additional field there that contains this flag, and you can call the enum name `REINDEX_TRANSACTION` or something similar.
I think that is cleaner towards future expansion, else we may potentially end up with a whole list of booleans.